src/components/policy/policy_external/src/policy_table/types.cc

```diff
@@ -1253,7 +1253,7 @@ Json::Value AppLevel::ToJsonValue() const {
   impl::WriteJsonField("count_of_removals_for_bad_behavior",
                        count_of_removals_for_bad_behavior,
                        &result__);
-  impl::WriteJsonField("count_of_TLS_errors", count_of_tls_errors, &result__);
+  impl::WriteJsonField("count_of_tls_errors", count_of_tls_errors, &result__);
```

Comment:
@jacobkeeler, verified the name with PT Data Dictionary. Proper naming is `count_of_TLS_errors`.
Fixed in 2d4aa21